planning/behavior_velocity_planner/include/utilization/trajectory_utils.hpp

```diff
@@ -100,7 +100,9 @@ TrajectoryPointWithIdx getLerpTrajectoryPointWithIdx(
     tier4_autoware_utils::calcLongitudinalOffsetToSegment(points, nearest_seg_idx, point);
   const double len_segment =
     tier4_autoware_utils::calcSignedArcLength(points, nearest_seg_idx, nearest_seg_idx + 1);
-  const double interpolate_ratio = std::clamp(len_to_interpolated / len_segment, 0.0, 1.0);
+  const double ratio = len_to_interpolated / len_segment;
+  if (ratio <= 0.0 || 1.0 <= ratio) return boost::none;
```

Comment:
I think this still does not prevent that the very close point is inserted such as the case that the ratio is 0.00001
Is it OK?

Reply:
@tkimura4 
that case is already covered here, so case of distance under 0.01 or 0.09 here is not insert point
https://github.com/autowarefoundation/autoware.universe/blob/bcfb8c5972661f640045ea29b35fae5b062ef2dd/planning/behavior_velocity_planner/include/utilization/trajectory_utils.hpp#L153